conda_build/build.py

```diff
@@ -599,7 +600,7 @@ def warn_on_old_conda_build(index=None, installed_version=None, available_packag
 """ % (installed_version, available_packages[-1]), file=sys.stderr)
 
 
-def build(m, config, post=None, need_source_download=True, need_reparse_in_env=False):
+def build(m, config, post=None, need_source_download=True, need_reparse_in_env=False, verify=True):
```

Comment:
verify should probably be a config parameter, rather than a new argument here.  Take a look at how test is done for reference.